src/Plugins/ResizeMirror/ResizeMirror.js

```diff
@@ -0,0 +1,93 @@
+import AbstractPlugin from 'shared/AbstractPlugin';
+
+const onDragOver = Symbol('onDragOver');
+
+/**
+ * ResizeMirror default options
+ * @property {Object} defaultOptions
+ * @type {Object}
+ */
+export const defaultOptions = {};
+
+/**
+ * ResizeMirror plugin resizes the mirror element based on the draggable elements it's hovering over
```

Comment:
does the mirror element's size depend on the particular draggable element that it's over? or does it change to a set size when it's over any draggable element

Reply:
The plugin is basically trying to find draggable elements you are hovering over and applying/copying the dimensions of the draggable element, that you are currently hovering over, onto the mirror.